resources/windows/msi/service.puppet.wxs

```diff
@@ -0,0 +1,73 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+  <Fragment>
+    <DirectoryRef Id="TBD">  # The ID has to match the Ruby Bin directory
+      <!--Seriously?... http://stackoverflow.com/questions/9419411/wix-setting-service-startup-type-using-a-property-property-not-recognized -->
+      <!-- Yep, Seriously - http://sourceforge.net/mailarchive/forum.php?thread_name=CANJN1a6gDE2tUUnP4SE05m8Ojh2q0Y8LrWOxbrF89YyJJCHn1A%40mail.gmail.com&forum_name=wix-users -->
+      <Component Id='PuppetServiceAutomatic' Guid="639ECD7F-6186-43D5-9E1A-FC0278DBEE15" Win64="$(var.Win64)">
+        <Condition>
+          <![CDATA[ (PUPPET_AGENT_STARTUP_MODE ~<> "manual") AND (PUPPET_AGENT_STARTUP_MODE ~<> "disabled") ]]>
+        </Condition>
+        <File Id="RubyExeAutomatic" KeyPath="yes" Source="stagedir\sys\ruby\bin\ruby.exe" />
+
+        <ServiceInstall Id="ServiceInstaller"
+          Account="[PUPPET_AGENT_ACCOUNT_DOMAIN]\[PUPPET_AGENT_ACCOUNT_USER]"
+          Password="[PUPPET_AGENT_ACCOUNT_PASSWORD]"
+          Description="Periodically fetches and applies configurations from a Puppet master server."
+          DisplayName="Puppet Agent"
+          Interactive="no"
+          Name="$(var.PuppetServiceName)"
```

Comment:
`$(var.PuppetServiceName)` isn't available if we're following through with not using `puppet.wxi` anymore